src/main/resources/schemas/V60__smartdocuments_informatie_object_type.sql

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+ALTER TABLE ${schema}.smartdocuments_document_creation_template
+    ADD informatie_object_type_url VARCHAR NOT NULL;
+
+COMMENT ON COLUMN ${schema}.smartdocuments_document_creation_template.informatie_object_type_url is 'URL of the Informatie Object Type';
+
+ALTER TABLE ${schema}.smartdocuments_document_creation_template
+    ADD CONSTRAINT un_sd_template_zaakafhandelparameters_id_informatie_object_type_url UNIQUE (zaakafhandelparameters_id, informatie_object_type_url);
```

Comment:
`un_sd_template_zaakafhandelparameters_id_informatie_object_type_url` might be too long for PostgreSQL? We already have one that is too long (which we should fix at some point I think):

```13:16:16,863 WARN  [org.flywaydb.core.internal.sqlscript.DefaultSqlScriptExecutor] (ServerService Thread Pool -- 28) DB: identifier "un_enkelvoudiginformatieobject_lock_uuid_enkelvoudiginformatieobject" will be truncated to "un_enkelvoudiginformatieobject_lock_uuid_enkelvoudiginformatieo" (SQL State: 42622 - Error Code: 0)```